### PR TITLE
[SLE-12-SP5 Preparation] Host upgrade from SLE-11-SP4 to SLE-12-SP5 needs the same treatment as to SLE-15

### PR DESCRIPTION
**Pull request by waynechen55**
1st time configuration step after upgrade is needed for SLE-12-SP5 as upgraded product

- Related ticket: 
  https://openqa.suse.de/tests/2952779
  https://openqa.suse.de/tests/2952674
- Needles: n/a
- Verification run: Might not be needed for this case

### CODE REVIEW

**Comment by okurz** [comment]
s/uprade/upgrade/

File: tests/virt_autotest/login_console.pm [hunk 1]
```diff
@@ -120,7 +120,7 @@ sub login_to_console {
             my $host_upgrade_version = get_required_var('UPGRADE_PRODUCT');         #format sles-15-sp0
             my $host_upgrade_relver  = $host_upgrade_version =~ /sles-(\d+)-sp/i;
             my $host_upgrade_spver   = $host_upgrade_version =~ /sp(\d+)$/im;
-            if (($host_installed_version eq '11') && ($host_upgrade_relver eq '15') && ($host_upgrade_spver eq '0')) {
+            if (($host_installed_version eq '11') && (($host_upgrade_relver eq '15' && $host_upgrade_spver eq '0') || ($host_upgrade_relver eq '12' && $host_upgrade_spver eq '5'))) {
```
Comment (okurz) [inline tests/virt_autotest/login_console.pm]: could you parse these version parts from the test variable VERSION? We should really try to have code that is not depending on the current version under test.

Comment (waynechen55) [reply tests/virt_autotest/login_console.pm]: They are already derived from 'VERSION', 'VERSION_TO_INSTALL' and  'UPGRADE_PRODUCT'.

**Comment by waynechen55** [comment]
> s/uprade/upgrade/

Done

**Comment by waynechen55** [comment]
I think this can be merged if no other concerns.

**Comment by waynechen55** [comment]
Please help merge this into master @alice-suse

**Pull request merged**